### ISSUE TITLE
Eclipse flags 37 maven build file issues

### ISSUE THREAD

**Issue by jimzucker** [capitalone/Hygieia#334]
This look pretty mechanical to clean up.  If we get eclipse clean it is easier to see when a new issue is raised.  Once Java8 is merged I will pull a branch and clean these up.

Description	Resource	Path	Location	Type
Duplicating managed version 1.1.1 for json-simple	pom.xml	/chat-ops-collector	line 93	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/core	line 127	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/github-scm-collector	line 93	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/jenkins-cucumber-test-collector	line 97	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/sonar-codequality-collector	line 94	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/udeploy-deployment-collector	line 77	Maven pom Loading Problem
Duplicating managed version 1.1.1 for json-simple	pom.xml	/versionone-feature-collector	line 81	Maven pom Loading Problem
Duplicating managed version 1.10.19 for mockito-core	pom.xml	/api	line 178	Maven pom Loading Problem
Duplicating managed version 1.10.19 for mockito-core	pom.xml	/core	line 140	Maven pom Loading Problem
Duplicating managed version 5.1.3.Final for hibernate-validator	pom.xml	/api	line 113	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/api	line 3	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/chat-ops-collector	line 5	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/core	line 6	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/jira-feature-collector	line 5	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/subversion-collector	line 4	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/udeploy-deployment-collector	line 4	Maven pom Loading Problem
GroupId is duplicate of parent groupId	pom.xml	/versionone-feature-collector	line 5	Maven pom Loading Problem
Overriding managed version 0.4.0 for docker-maven-plugin	pom.xml	/UI	line 85	Maven pom Loading Problem
Overriding managed version 0.9.1 for json-path	pom.xml	/api	line 190	Maven pom Loading Problem
Overriding managed version 1.1.3 for logback-classic	pom.xml	/api	line 146	Maven pom Loading Problem
Overriding managed version 1.1.3 for logback-classic	pom.xml	/core	line 84	Maven pom Loading Problem
Overriding managed version 1.8.0.RELEASE for spring-data-mongodb	pom.xml	/core	line 104	Maven pom Loading Problem
Overriding managed version 2.5 for joda-time	pom.xml	/api	line 165	Maven pom Loading Problem
Overriding managed version 2.5.0 for jackson-databind	pom.xml	/api	line 127	Maven pom Loading Problem
Overriding managed version 3.1 for maven-compiler-plugin	pom.xml	/api	line 44	Maven pom Loading Problem
Overriding managed version 3.1 for maven-compiler-plugin	pom.xml	/core	line 49	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-test	pom.xml	/core	line 98	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/chat-ops-collector	line 87	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/github-scm-collector	line 87	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/jenkins-cucumber-test-collector	line 85	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/sonar-codequality-collector	line 88	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/udeploy-deployment-collector	line 71	Maven pom Loading Problem
Overriding managed version 4.1.7.RELEASE for spring-web	pom.xml	/versionone-feature-collector	line 75	Maven pom Loading Problem
Overriding managed version 4.11 for junit	pom.xml	/core	line 134	Maven pom Loading Problem
Overriding managed version 4.12 for junit	pom.xml	/api	line 172	Maven pom Loading Problem
Overriding managed version 5.1.3.Final for hibernate-validator	pom.xml	/core	line 163	Maven pom Loading Problem

**Comment by jimzucker** [comment]
Looks like all of them are 'noise' except jodi-time needs to be upgraded 2.5->2.7 (api/pom.xml has 2.7)

**Comment by tabladrum** [comment]
Simple to clean up. Will do.

**Comment by jimzucker** [comment]
@tabladrum Ill take care of them i am just waiting for the java 1.8 pull req to be merged. (I am going thru cleaning up eclipse and wiill create a ticket for each category of cleanups to give the team visibility. Please assign to me.

Sent from my iPhone

On Feb 19, 2016, at 06:24, tabladrum <notifications@github.com<mailto:notifications@github.com>> wrote:


Simple to clean up. Will do.

-
